snapcraft/plugins/python2.py

```diff
@@ -40,182 +40,35 @@
       A list of dependencies to get from PyPi
 """
 
-import glob
 import logging
 import os
-import re
 
-import snapcraft
-from snapcraft import common
+from snapcraft.plugins import python3
 
 logger = logging.getLogger(__name__)
 
 
-class Python2Plugin(snapcraft.BasePlugin):
+class Python2Plugin(python3.Python3Plugin):
```

Comment:
this is not a great idea, conceptually. Python3 is not a parent of python2, so inheritance could cause problems in the future. A better approach might be to make a common BasePythonPlugin, and inherit both Python2 and Python3 plugins from it.